src/commands/branches.ts

```diff
@@ -189,17 +189,21 @@ export const builder = (argv: yargs.Argv) =>
       async (args) => await get(args as any),
     )
     .command({
-      command: 'schema-diff [base-branch] <compare-branch>',
+      command: 'schema-diff [base-branch] [compare-source]',
       aliases: ['sd'],
-      describe: 'Compare schemas from two branches',
-      builder: (yargs) =>
-        yargs.options({
+      describe:
+        'compare a branch schema to a specific point in time' +
+        '\n  [compare-source] can be: ^self, ^parent, or <compare-branch-id|name>',
```

Comment:
I feel that real power of this addition is not really covered in the description.
I'd suggest doing it similar to "restore" command - writing text extensive format that includes "@" and adding examples

Reply:
Update the command to include `@timestamp|lsn` and added examples.